content/technotes/2020-03-19-parzer/index.Rmd

```diff
@@ -0,0 +1,166 @@
+---
+slug: "parzer"
+title: "parzer: Parse Messy Geographic Coordinates"
+date: 2020-03-19
+author:
+  - Scott Chamberlain
+description: parzer, a new package for parsing messy geographic coordinates.
+tags:
+  - geospatial
+  - parzer
+  - software peer review
+  - R
+output: 
+  md_document:
+    variant: gfm
```

Comment:
Ah @sckott this might be why the shortcodes disappeared even with htmlpreserve. In the Rmd template linked from the guidance the output is an html document with keep_md TRUE because that was the only way to get the Md we needed (and the html is gitignored and Hugo ignored anyway)

Reply:
ah okay